### ISSUE TITLE
AccessibilityValue missing from list pages

### ISSUE THREAD

**Issue by bessgreen** [department-of-veterans-affairs/va.gov-team#22356]
## What happened?
AccessibilityValues are not being announced on the following pages in the app:
     - Appointments list pages (Upcoming/Past)
     - Claims & Appeals list pages (Active/Closed)

We should be using the accessibilityValue for each ListItem so that the item's position in the list (i.e. "2 of 6") is announced.

## Specs:
- Device: iPhone 11 (VoiceOver), Samsung Galaxy S9 (Voice Assistant)

## Steps to Reproduce
1. Enable the device's screen reader tool
2. Select ListItems on the Appointments and Claims/Appeals list pages
3. Observe that the announcement does not include any reference to the item's position in the list

## Desired behavior
Items that are part of a list should include an announcement of the item's position in the list.

## Acceptance Criteria
1. ListItems on the following pages include an accessibilityValue that announces the item's position in the list:
       - Appointments list pages (upcoming/past)
       - Claims & Appeals list pages (active/closed)

## Bug Severity
- 3 - Medium

## Linked to Story
#13971
#13994

## Screen shot(s) and additional information
N/A

**Comment by digitalken** [comment]
PASS 6/18/2021 - Sounds good on iOS and Android. Closing.
